src/main/scala/ch/openolitor/stammdaten/StammdatenUpdateService.scala

```diff
@@ -921,4 +940,13 @@ class StammdatenUpdateService(override val sysConfig: SystemConfig) extends Even
       }
     }
   }
+
+  private def getUpdatedDescriptionLieferplanung(abotypDepotTour: String, newName: String, oldName: String): String = {
```

Comment:
This logic must have been somewhere beforehand, no?

Reply:
There are some helper functions in this service that is why I decided to place this function here (f.i. adjustGuthabenVorLieferung). However I can create a handler trait for the lieferplanung. this is what you meant??